api/interestr/api/serializers.py

```diff
@@ -34,13 +34,64 @@ class Meta:
 
 class DataTemplateSimpleSerializer(serializers.ModelSerializer):
 
+    VALID_TYPES = [
+        'checkbox',
+        'multisel',
+        'textarea',
+        'text',
+        'number',
+        'date',
+        'email',
+        'url',
+        'tel'
+    ]
```

Comment:
List of valid types does not have the newly introduced `"select"` type of fields. Since this pull request comes after the #174, it should include this.

Reply:
Ok